src/legacy/core_plugins/telemetry/public/components/opt_in_message.tsx

```diff
@@ -21,8 +21,7 @@ import * as React from 'react';
 import { EuiLink } from '@elastic/eui';
 import { FormattedMessage } from '@kbn/i18n/react';
 
-import { getConfigTelemetryDesc, PRIVACY_STATEMENT_URL } from '../../common/constants';
-import { OptInExampleFlyout } from './opt_in_details_component';
```

Comment:
Looks like `opt_in_details_component.ts` and the test file are still in the code, but can be removed.

Reply:
nm - it's still used from the Advanced Settings form for telemetry

Reply:
I think that code is used by some other code that is still around - a license management page?  Could be dead code by the time we're done with all the changes, but didn't look like I could just remove the files right now.